pages.es/common/git-rebase.md

```diff
@@ -4,9 +4,9 @@
 > Se utiliza comúnmente para "mover" una rama entera a otra base, ya que crea copias de los commits en una nueva ubicación.
 > Más información: <https://git-scm.com/docs/git-rebase>.
 
-- Rebasa la rama actual en lo más alto de la rama master:
+- Rebasa la rama actual en lo más alto de una rama:
 
-`git rebase {{master}}`
+`git rebase {{branch_name}}`
```

Comment:
It's unclear whether `branch_name` refers to the current branch or the target branch. We might want to use a more explicit token name here.

Reply:
```suggestion
`git rebase {{rama_base}}`
```

Reply:
In English, I'd go with `target_branch` or `new_base_branch`. That's because the very notion of rebasing implies two bases, the old one and the new one, so merely base_branch would still be ambiguous as it could refer to either base. How would you suggest we express this concept in Spanish?

Reply:
Re-reading again [the docs](https://git-scm.com/book/es/v2/Ramificaciones-en-Git-Reorganizar-el-Trabajo-Realizado) they use the term:
```suggestion
`git rebase {{rama_de_reorganización}}`
```